sources/web/datalab/polymer/components/datalab-toolbar/datalab-toolbar.html

```diff
@@ -115,7 +116,9 @@
     <!--Settings dialog. This shows centered in the middle of the window-->
     <paper-dialog id="settingsDialog" class="dialog" with-backdrop>
```

Comment:
Why? The `datalab-settings` element can theoretically be embedded in any container. For example, we might want to switch to have the settings be its own page instead of a dialog.